tests/rust/assoc_types.rs

```diff
@@ -0,0 +1,41 @@
+
+pub const PI: f64 = 3.1415926535;
```

Comment:
do we need concrete type? it should have been tested in other tests

Reply:
No, I added it as a sanity test during my testing to make sure I didn't accidentally break anything else